ports/stm32/boards/startup_stm32f7.s

```diff
@@ -243,6 +243,8 @@ g_pfnVectors:
   .word     FPU_IRQHandler                    /* FPU                          */
   .word     UART7_IRQHandler                  /* UART7                        */
   .word     UART8_IRQHandler                  /* UART8                        */
+
+#if defined(MCU_SERIES_F7)
```

Comment:
This shouldn't be needed.  There is an individual file for each of F4/F7/L4/H7, so they no longer need these macro checks.

Reply:
Whoops, I wrote the patch on 1.9.3 and didn't catch that while rebasing